pkg/Library/Graph/Atoms/Layer.js

```diff
@@ -4,7 +4,7 @@ export const atom = (log, resolve) => ({
  * Copyright 2023 NeonFlan LLC
  * SPDX-License-Identifier: BSD-3-Clause
  */
-async update({graphId, graph, data, designable}, state, {service, isDirty, output}) {
+async update({graphId, graph, data, designable, composer}, state, {service, isDirty, output}) {
```

Comment:
nit - should the input maybe be called `composerId`? i understand that the actual composer won't be passed as input... but it might still be clearer from the naming perspective